dig.go

```diff
@@ -229,7 +196,7 @@ func New(opts ...Option) *Container {
 		values:    make(map[key]reflect.Value),
 		groups:    make(map[key][]reflect.Value),
 		rand:      rand.New(rand.NewSource(time.Now().UnixNano())),
-		dg:        new(dotGraph),
+		dg:        new(dot.Graph),
```

Comment:
It may be safer in the long run to build `dot.Graph` on-demand from a `Container` rather than keeping a running record of it. Otherwise any changes to internal book-keeping have to be propagated to the `dot.Graph` code.

Reply:
Good point, we can address this in a future PR